windows/Dockerfile

```diff
@@ -1,6 +1,7 @@
-FROM microsoft/nanoserver:sac2016
+FROM mcr.microsoft.com/windows/nanoserver:1809
 
 COPY ./traefik.exe /traefik.exe
+COPY ./netapi32.dll /windows/system32/netapi32.dll
```

Comment:
This file should be downloaded in the same manner as `traefik.exe` is, so it can be updated when the `update*.sh` scripts are called.